test/jobs/fetch_faculty_data_job_test.rb

```diff
@@ -71,7 +71,6 @@ def expected_building_names
         "Crean Hall",
         "DeMille Hall",
         "Digital Media Arts Center",
-        "Dodge College",
```

Comment:
I just fixed the existing tests, but to be honest these faculty tests aren't very pretty to me. A lot of them are hardcoded to make assumptions about the live data. I don't really think that they should be behaving like that as (made clear by this issue even existing) the live data is susceptible to change.